dali/operators/image/remap/remap.cu

```diff
@@ -61,6 +64,27 @@ class RemapGpu : public Remap<GPUBackend> {
                {}, {}, make_span(interps_), {});
   }
 
+
+  /**
+   * Stores and returns a NppRemapKernel object. Creates one if it doesn't exist.
+   */
+  template<typename InputType>
+  kernels::remap::NppRemapKernel<StorageGPU, InputType> &GetOrCreateKernel(int device_id) {
+    assert(device_id >= 0);
+    static std::unordered_map<int /* device_id */,
```

Comment:
Why static variable inside a function instead of making it a field of a kernel (or some other place).
Are we sure that this function won't be called from multiple threads?

Reply:
Don't we have Kernel manager for something like that?

Reply:
:+1: